lib/trento/application/usecases/sap_systems/health_summary_service.ex

```diff
@@ -48,10 +48,26 @@ defmodule Trento.SapSystems.HealthSummaryService do
       sapsystem_health: health,
       database_health: compute_database_health(database_instances),
       clusters_health: compute_clusters_health(all_instances),
-      hosts_health: compute_hosts_health(all_instances)
+      hosts_health: compute_hosts_health(all_instances),
+      hana_cluster_id: extract_hana_cluster_id(database_instances),
+      database_id: extract_database_id(database_instances)
     })
   end
 
+  @spec extract_database_id([DatabaseInstanceReadModel.t()]) :: String.t()
+  defp extract_database_id(database_instances) do
+    database_instances
+    |> Enum.map(fn %{sap_system_id: database_id} -> database_id end)
+    |> List.first()
+  end
+
+  @spec extract_hana_cluster_id([DatabaseInstanceReadModel.t()]) :: String.t()
+  defp extract_hana_cluster_id(database_instances) do
```

Comment:
Just in case we change the `hana_cluster_id` the function should be renamed

Reply:
Keep in mind, that we could have a non clustered database, which in this case, we wouldn't get anything.
I don't know if this could break some pattern matching rule. We would be get a nil value most probably.

Reply:
This choice was made because the readmodel is directly consumed by the frontend and we have a different naming on the frontend for these things. Anyway no strong opinions, I could change it :D

Reply:
Removed the hana_ naming stuff